packages/compiler/src/emitter-framework/types.ts

```diff
@@ -57,6 +57,9 @@ export interface AssetEmitter<T, TOptions extends object = Record<string, unknow
     none(): NoEmit;
   };
   writeOutput(): Promise<void>;
+
+  // TODO - is is a good way?????
+  getSourceFiles(): SourceFile<T>[];
```

Comment:
Added this so I could collect what was emitted myself here, is that an ok way

Reply:
Seems fine to me!